demo/src/main/java/com/orange/ods/demo/ui/components/buttons/ButtonsContained.kt

```diff
@@ -18,34 +18,42 @@ import androidx.compose.ui.res.dimensionResource
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import com.orange.ods.compose.component.button.OdsButton
-import com.orange.ods.compose.component.button.OdsButtonContainedStyle
+import com.orange.ods.compose.component.button.OdsButtonStyle
 import com.orange.ods.compose.theme.OdsDisplaySurface
 import com.orange.ods.demo.R
+import com.orange.ods.demo.ui.utilities.composable.TechnicalText
 import com.orange.ods.demo.ui.utilities.composable.Title
 
 @Composable
-fun ButtonsContained() {
-    Title(R.string.component_buttons_contained_primary, withHorizontalPadding = true)
-    ContainedButtons(style = OdsButtonContainedStyle.Primary)
+fun ButtonsHighEmphasis() {
+    Title(R.string.component_buttons_high_emphasis_primary, withHorizontalPadding = true)
+    TechnicalText(text = "OdsButtonStyle.Primary", withHorizontalPadding = true)
```

Comment:
We can add an extension on `Enum`:

```kotlin
val Enum<*>.fullName: String
    get() = "${javaClass.simpleName}.$name"
```

and use it here to avoid typos:

```kotlin
TechnicalText(text = OdsButtonStyle.Primary.fullName, withHorizontalPadding = true)
```